src/Apps/Collections/Components/CollectionsGrid.tsx

```diff
@@ -1,52 +1,63 @@
 import { Box, Flex, Sans, Separator } from "@artsy/palette"
-import { CollectionsApp_collections } from "__generated__/CollectionsApp_collections.graphql"
 import React, { Component } from "react"
 import { EntityHeader } from "Styleguide/Components/EntityHeader"
-import { Responsive } from "Utils/Responsive"
+import { slugify } from "underscore.string"
+import { crop } from "Utils/resizer"
+import { Media } from "Utils/Responsive"
+
+export interface CollectionEntity {
+  title: string
+  headerImage?: string
+  slug: string
+}
 
 interface CollectionsGridProps {
-  collections: CollectionsApp_collections
+  collections: CollectionEntity[]
   name?: string
 }
 
 export class CollectionsGrid extends Component<CollectionsGridProps> {
   render() {
     const { collections, name } = this.props
+    const hasShortRow = collections.length % 3 !== 0 // Preserve left align
 
     return (
-      <Responsive>
-        {({ xs }) => (
-          <Box>
-            <Sans size="3" weight="medium">
-              {name}
-            </Sans>
-
-            <Flex flexWrap="wrap" justifyContent="space-between">
-              {collections.map((collection, index) => {
-                const hasBorderTop = xs ? index === 0 : index < 3
-
-                return (
-                  <Flex
-                    width={xs ? "100%" : "30%"}
-                    flexDirection="column"
-                    key={index}
-                  >
-                    {hasBorderTop && <Separator />}
-                    <Box py={2}>
-                      <EntityHeader
-                        href={`/collection/${collection.slug}`}
-                        imageUrl={collection.headerImage}
-                        name={collection.title}
-                      />
-                    </Box>
-                    <Separator />
-                  </Flex>
-                )
-              })}
-            </Flex>
-          </Box>
-        )}
-      </Responsive>
+      <Box pb={80} id={slugify(name)}>
+        <Sans size="3" weight="medium" pb={15}>
+          {name}
+        </Sans>
+
+        <Flex flexWrap="wrap" justifyContent="space-between">
+          {collections.map((collection, index) => {
+            const imageUrl =
+              collection.headerImage &&
+              crop(collection.headerImage, {
```

Comment:
I would not set a width here to preserve the correct aspectRatio

Reply:
The Entity component sizes the image to 50x50, and expects a square file -- otherwise they are squeezed to a square: https://artsy.slack.com/archives/CD0B3F620/p1542210908007300?thread_ts=1542146808.001600&cid=CD0B3F620